.github/workflows/cla.yml

```diff
@@ -0,0 +1,26 @@
+name: 'CLA Assistant'
+on:
+  issue_comment:
+    types: [created]
+  pull_request_target:
+    types: [opened, closed, synchronize]
+
+jobs:
+  CLAssistant:
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'CLA Assistant'
+        if: (github.event.comment.body == 'recheckcla' || github.event.comment.body == 'I have read the CLA Document and I hereby sign the CLA') || github.event_name == 'pull_request_target'
+        # Alpha Release
+        uses: gnosis/cla-github-action@master
+        # GitHub token, automatically provided to the action
+        # (No need to define this secret in the repo settings)
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          path-to-signatures: 'signatures/version1/cla.json'
+          path-to-cla-document: 'https://gnosis-safe.io/cla/'
+          branch: 'cla-signatures'
+          allowlist: jpalvarezl,rmeissner,Uxio0,bot*,fmrsabino
```

Comment:
If `bot*` is a wildcard I'd be more conservative and only list the ones that we have enabled (`dependabot`)

Reply:
It seems to be a reserved name: https://github.com/cla-assistant/github-action#1-add-the-following-workflow-file-to-your-repository-in-this-pathgithubworkflowsclayml

Reply:
I read the following:

> And also, since there's no way for bot users (such as Dependabot or Greenkeeper) to sign a CLA, you may want to add them in allowlist. You can do so by adding their names in a comma separated string to the allowlist input in the CLA workflow file(in this case dependabot-preview[bot],greenkeeper[bot]). You can also use wildcard symbol in case you want to allow all bot users something like bot*.

So I thought that `bot*` would not work with dependant but maybe it does 👍

Reply:
hahah sorry 🙇 ! you are totally right, I just saw it in the default docs like that too and I assued it. I am wondering how does dependabot match in the safe-apps-sdk repo then 🤔

Reply:
Its username is: `dependabot[bot]`. Maybe we can just add that.